Sources/PixelExperimentKit/PixelExperimentKit.swift

```diff
@@ -94,11 +94,43 @@ extension PixelKit {
         }
         guard let experimentData = featureFlagger.getAllActiveExperiments()[subfeatureID] else { return }
 
+        // Define event
+        let event = event(for: subfeatureID, experimentData: experimentData, conversionWindowDays: conversionWindowDays, metric: metric, value: value)
+
+        // Send unique by name and parameter pixel if within conversion window
+        let isInWindow = isUserInConversionWindow(conversionWindowDays, enrollmentDate: experimentData.enrollmentDate)
+        if isInWindow {
+            ExperimentConfig.fireFunction(event, .uniqueByNameAndParameters, false)
+        }
```

Comment:
Minor: We could avoid the creation of the event and return early by moving the conversion window check up:

```suggestion
        // Check if within conversion window
        guard isUserInConversionWindow(conversionWindowDays, enrollmentDate: experimentData.enrollmentDate) else { return }

        // Define event
        let event = event(for: subfeatureID, experimentData: experimentData, conversionWindowDays: conversionWindowDays, metric: metric, value: value)

        ExperimentConfig.fireFunction(event, .uniqueByNameAndParameters, false)
```